Makefile

```diff
@@ -454,3 +454,38 @@ builds/torus-npm-$(VERSION).tar.gz: npm
 	tar czf $@ -C builds npm/
 
 .PHONY: npm
+
+
+#################################################
+# Bootstrap Windows
+#################################################
+
+WIX := $(shell command -v torch 2> /dev/null)
+GOMSI := $(shell command -v go-msi 2> /dev/null)
+
+bootstrap-windows: bootstrap
+ifndef WIX 
+	curl --create-dirs -fsSL -k -o C:\Temp\wix311-binaries.zip https://github.com/wixtoolset/wix3/releases/download/wix311rtm/wix311-binaries.zip
+	unzip "C:\Temp\wix311-binaries.zip" -d "C:\Program Files\go-msi"
+endif
+ifndef GOMSI 
+	curl --create-dirs -fsSL -k -o C:\Temp\go-msi.msi http://github.com/mh-cbon/go-msi/releases/download/1.0.2/go-msi-amd64.msi
+	msiexec /i C:\Temp\go-msi.msi /q
+endif
+.PHONY: bootstrap-windows
+	
+#################################################
+# Generate Windows .msi 
+#################################################
+
+msi: binary 
```

Comment:
I think this should depend on the `release-binary-windows` rule? This way it also gets put inside the `builds/` directory. The subsequent msi should probably also be placed there as well, so it gets deleted on `make clean`.